test/message/test_runner_output.js

```diff
@@ -61,6 +61,11 @@ test('async assertion fail', async () => {
   assert.strictEqual(true, false);
 });
 
+test('async assertion object fail', async () => {
```

Comment:
Maybe something like:

```suggestion
test('assertion errors display actual and expected properly', async () => {
```